async-nats/examples/jetstream_kv_get_put_delete.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020-2022 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::env;
+use std::str;
+
+use async_nats::jetstream;
+
+#[tokio::main]
```

Comment:
You're using `Result`, yet you're `unwrapping` throughout the function. 
Please leverage the `Result`.